content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,25 +9,82 @@ weight: 20
 
-Kubernetes generally leverages standard RESTful terminology to describe the API concepts:
+You can view the [API reference](/docs/reference/kubernetes-api/) online,
+or read on to learn about the API in general.
 
-* A **resource type** is the name used in the URL (`pods`, `namespaces`, `services`)
-* All resource types have a concrete representation in JSON (their object schema) which is called a **kind**
-* A list of instances of a resource type is known as a **collection**
-* A single instance of the resource type is called a **resource**
-
-All resource types are either scoped by the cluster (`/apis/GROUP/VERSION/*`) or to a namespace (`/apis/GROUP/VERSION/namespaces/NAMESPACE/*`). A namespace-scoped resource type will be deleted when its namespace is deleted and access to that resource type is controlled by authorization checks on the namespace scope. The following paths are used to retrieve collections and resources:
+<!-- body -->
+## Kubernetes API terminology {#standard-api-terminology}
+
+Kubernetes generally leverages common RESTful terminology to describe the
+API concepts:
+
+* A *resource type* is the name used in the URL (`pods`, `namespaces`, `services`)
+* All resource types have a concrete representation (their object schema) which is called a *kind*
+* A list of instances of a resource is known as a *collection*
+* A single instance of a resource type is called a *resource*, and also usually represents an *object*
+* For some resource types, the API includes one or more *sub-resources*, which are represented as URI paths below the resource
+
+Most Kubernetes API resource types are
+[objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects):
+they represent a concrete instance of a concept on the cluster, like a
+pod or namespace. A smaller number of API resource types are *virtual* in
+that they often represent operations on objects, rather than objects, such
+as a permission check
+(use a POST with a JSON-encoded body of `SubjectAccessReview` to the
+`subjectaccessreviews` resource), or the `eviction` sub-resource of a Pod
+(used to trigger
+[API-initiated eviction](/docs/concepts/scheduling-eviction/api-eviction/)).
+
+### Object names
+
+All objects you can create via the API have a unique object
+{{< glossary_tooltip text="name" term_id="name" >}} to allow idempotent creation and
+retrieval, except that virtual resource types may not have unique names if they are
+not retrievable, or do not rely on idempotency.
+Within a {{< glossary_tooltip text="namespace" term_id="namespace" >}}, only one object
+of a given kind can have a given name at a time. However, if you delete the object,
+you can make a new object with the same name. Some objects are not namespaced (for
+example: Nodes), and so their names must be unique across the whole cluster.
+
+### API verbs
+
+Almost all object resource types support the standard HTTP verbs - GET, POST, PUT, PATCH,
```

Comment:
I have a question about: "Almost all object resource types". Is this the same as:
A Kubernetes resource type that represents an object supports the HTTP verbs: GET, POST, ...

When are the Kubernetes verbs (lowercase) used (add this information)?

Reply:
> When are the Kubernetes verbs (lowercase) used?

Could you clarify that question? I infer you're asking on behalf of the reader. What is it that they want to know?